internal/login/command.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	ccloudv1 "github.com/confluentinc/ccloud-sdk-go-v1-public"
 
-	"github.com/confluentinc/cli/v3/internal/admin"
+	"github.com/confluentinc/cli/v3/internal/billing"
```

Comment:
Nit: We should avoid calling `internal` packages from other `internal` packages to avoid import cycle issues. Consider moving the `ConvertToUsd()` function to `pkg`?

Reply:
Done. We already had a `billing` pkg so I moved it there.